tests/bootstrap.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$command = 'php -S localhost:8000 -t ./web >/dev/null 2>&1 & echo $!';
```

Comment:
composer.json requires a minimum PHP version of 5.3, but this feature was only introduced in 5.4